rust/noosphere-ipfs/src/kubo.rs

```diff
@@ -83,7 +93,11 @@ impl IpfsClient for KuboClient {
         }
     }
 
-    async fn get_block(&self, cid: &Cid) -> Result<Vec<u8>> {
+    async fn put_block(&mut self, _cid: &Cid, _block: &[u8]) -> Result<()> {
+        unimplemented!();
```

Comment:
TBD the differences between `put_block` and `syndicate_blocks` here, though `IpfsStore` only uses `get_block`.